src/components/Expenses.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Template from './Template.js';
+
+const Wrapper = () => {
```

Comment:
It seems like you are essentially passing an empty div into template. Is it possible to eliminate the need or refactor the wrapper function ?